test/unittests/eof_test.cpp

```diff
@@ -38,9 +40,16 @@ TEST(eof, read_valid_eof1_header)
     for (int i = 0; i < 255; ++i)
         nops_255 += "5B";
 
-    std::string section_size_1_256;
+    std::string section_size_256;
     for (int i = 0; i < 256; ++i)
-        section_size_1_256 += "0001";
+        section_size_256 += "0003";
+
+    std::string code_sections_256;
+    for (int i = 0; i < 256; ++i)
+        if (i < 255)
+            code_sections_256 += "E5" + std::format("{:04X}", i + 1);
```

Comment:
I would probably do something like
```suggestion
            code_sections_256 += "E5" + hex(big_endian(static_cast<uint16>(i + 1)));
```

Reply:
It looks also some CI builds with old compilers fail because `std::format` is not available.